### PR TITLE
Upgrade from 1.44 to 1.45

### PR DESCRIPTION
**Pull request by crramirez**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19574)

### CODE REVIEW

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
You need to rename the folder from 1.4 to 1.45

File: manifests/a/AltDrag/AltDrag/1.4/AltDrag.AltDrag.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ PackageIdentifier: AltDrag.AltDrag
 Publisher: AltDrag
```
Comment (OfficialEsco) [inline manifests/a/AltDrag/AltDrag/1.4/AltDrag.AltDrag.yaml:2]:
```suggestion
Publisher: Raymond Gillibert
```
image

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Aaaalso the Publisher is incorrect

**Comment by crramirez** [comment]
Thanks a lot I was looking in the documentation how is the procedure to upgrade a package.

**Comment by OfficialEsco** [comment]
> Thanks a lot I was looking in the documentation how is the procedure to upgrade a package.

When its a static permanent link you can just make a new Manifest, when its a dynamic link you can rename and or delete the previous manifest and it will count as update and rename

Publisher/Software/Version is the folder structure, you can change the Version folder, but not the Publisher or Software folder.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**